### PR TITLE
 Add blog post for 1.5.0

### PR DESCRIPTION
**Pull request by thehowl**

### CODE REVIEW

**Comment by thehowl** [comment]
Should be good to go.

File: content/post/release-of-1.5.0.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
+---
+date: "2018-07-07T10:00:00+00:00"
+author: "thehowl"
+title: "Gitea 1.5.0 is released"
+tags: ["release"]
+---
+
+The time has come for another minor release! We are happy to present Gitea
```
[the remark this answers is not in the document]
Comment (thehowl) [reply content/post/release-of-1.5.0.md]: Wrong. Major bumps are those of the first number. https://semver.org/

File: content/post/release-of-1.5.0.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+  ([#3875](https://github.com/go-gitea/gitea/pull/3875))
+* If you want to disable time tracking entirely, you can now do so from the app
+  settings. ([#3719](https://github.com/go-gitea/gitea/pull/3719))
+* Various changes to improve consistency and grammar in the English
+  localisation.
+  ([Various](https://github.com/go-gitea/gitea/pulls?q=is%3Apr+author%3Abugreport0+is%3Aclosed+milestone%3A1.5.0))
+* You can now sort repos in Explore and the admin panel by stars or forks.
+  ([#3969](https://github.com/go-gitea/gitea/pull/3969))
+* Starting from 1.5.0, we'll sign all our releases with our
+  [GPG Key,](http://pool.sks-keyservers.net/pks/lookup?op=get&hash=on&fingerprint=on&search=0x2D9AE806EC1592E2)
```
Comment (JonasFranzDEV) [inline content/post/release-of-1.5.0.md]: http://pool.sks-keyservers.net/pks/lookup?op=get&hash=on&fingerprint=on&search=0x2D9AE806EC1592E2 returns 404 for me

File: content/post/release-of-1.5.0.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (techknowlogick) [reply content/post/release-of-1.5.0.md]: If we were adhering 100% to semver then yes, however this is considered a major release in Gitea versioning.

File: content/post/release-of-1.5.0.md (hunk 2 above)
Comment (sondr3) [reply content/post/release-of-1.5.0.md]: Seems like it's fixed now, I get a GPG key when I go to that URL.

**Comment by JonasFranzDEV** [comment]
Maybe we should mention that it is since 1.5 possible to deploy releases (and attachments) via drone to Gitea.

http://plugins.drone.io/drone-plugins/drone-gitea-release/

File: content/post/release-of-1.5.0.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+  so you can be sure it's us.
+
+Please see full changelog below for more details.
+
+A shoutout goes to those who reported and/or fixed security issues in this
+release:
+
+* [@cezar97](https://github.com/cezar97) ([#3878](https://github.com/go-gitea/gitea/pull/3878))
+
+**Deprecation notice:** in the upcoming minor release (1.6.0) we will drop
```
[the remark this answers is not in the document]
Comment (bkcsoft) [reply content/post/release-of-1.5.0.md]: 1.5 => 1.6 is a minor release if we follow SemVer. Major.Minor.Patch :thinking:

File: content/post/release-of-1.5.0.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lunny) [reply content/post/release-of-1.5.0.md]: see https://golang.org/project/ , golang said 1.0 1.1 as major release.
image
Or maybe we can rename 1.6 -> 6.0 ? :)

**Comment by thehowl** [comment]
updated

**Pull request merged**